my-vue3-app/src/store/reports/state.ts

```diff
@@ -10,15 +10,23 @@ export interface Report {
 
 export interface State {
   [REPORTS]: Array<Report>;
-  [PAGINATION]: {
+  [PAGINATION]: Pagination;
+}
+
+export interface Pagination {
     page: number;
     pageSize: number;
     itemsTotal: number;
-  };
 }
 
 export default {
-  [REPORTS]: [],
+  [REPORTS]: [
+    {
+      title: 'Title',
```

Comment:
remove test data

Reply:
Should there be some initial data for first report load?